internal/tools/tool/tool_darwin.go

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package tool
+
+import (
+	"bytes"
+	"os/exec"
+)
+
+func (t *tool) generateCommandLine(args []string) *exec.Cmd {
+	path, _ := exec.LookPath("open")
+
+	args = append([]string{"--args"}, args...)
+	args = append([]string{t.exeName}, args...)
+	args = append([]string{"-a"}, args...)
+
+	var stdout, stderr bytes.Buffer
+	cmd := &exec.Cmd{
+		Path:   path,
+		Args:   args,
+		Stdout: &stdout,
+		Stderr: &stderr,
+	}
+	fmt.Println(cmd)
```

Comment:
>fmt.Println(cmd) [](http://example.com/codeflow?start=1&length=16)

is this println a debugging aid, to be removed?

Reply:
yes!  I don't have a debugger set up on my Mac